tracing-subscriber/src/fmt/mod.rs

```diff
@@ -971,9 +808,11 @@ mod test {
     }
 
     #[test]
-    #[cfg(feature = "registry_unstable")]
+    #[cfg(feature = "registry")]
```

Comment:
if the "fmt" feature now requires the registry feature, which I think it does, we can remove this...